src/pageLayout/containers/MainNavigation.tsx

```diff
@@ -302,6 +309,12 @@ export const MainNavigation: FC = () => {
     event('helpBar.contactSupportRequest.overlay.shown')
   }
 
+  const handleRequestPocClick = () => {
+    event('nav.requestPOC.clicked')
+    const newTab = window.open('https://www.influxdata.com/proof-of-concept/')
```

Comment:
You may want to use `safeBlankLinkOpen` for security reason

https://github.com/influxdata/ui/blob/63139c54da890df5a3c2c3ca6754b54cf94e0348/src/utils/safeBlankLinkOpen.ts#L3